lib/model/model.go

```diff
@@ -1725,10 +1736,41 @@ func (m *model) SetIgnores(folder string, content []string) error {
 		return err
 	}
 
-	if err := ignore.WriteIgnores(cfg.Filesystem(), ".stignore", content); err != nil {
```

Comment:
We should really make sure we are writing the same file we are claiming to write in the GUI... Need to think about how to best do that...

Reply:
Before writing at this level, there are different conditions that check if we are actually writing in the file we are trying to write in Gui.